### PR TITLE
Update Crashpad from Chromium 84.0.4147.144

### PR DESCRIPTION
**Pull request by dahlstrom-g**
b/152473909

### CODE REVIEW

**Comment by hlwarriner** [comment]
Sorry for the delay here - I'll finish a first pass of this by this coming Monday.

File: third_party/crashpad/third_party/googletest/BUILD.gn [hunk 1]
```diff
@@ -0,0 +1,372 @@
+# Copyright 2017 The Crashpad Authors. All rights reserved.
```
Comment (hlwarriner) [inline third_party/crashpad/third_party/googletest/BUILD.gn:1]: I guess there's not a good way to show this file as branched from `third_party/crashpad/third_party/gtest/BUILD.gn`?

File: third_party/crashpad/minidump/BUILD.gn [hunk 2]
```diff
@@ -93,6 +93,25 @@ static_library("minidump") {
...
+
+  deps = [
+    "../snapshot:context",
+    "../third_party/mini_chromium:base",
+    "../util",
+  ]
+}
+
 static_library("test_support") {
   testonly = true
```
Comment (hlwarriner) [inline third_party/crashpad/minidump/BUILD.gn:116]: This `testonly` was a Cobalt customization that was removed in https://lbshell-internal-review.git.corp.google.com/c/cobalt_src/+/264151. But I guess the new version of Chromium had it?

File: third_party/crashpad/client/BUILD.gn [hunk 3]
```diff
@@ -147,7 +147,7 @@ source_set("client_test") {
   }
 
   if (crashpad_is_ios) {
-    sources += [ "crashpad_client_ios_test.cc" ]
```
[the remark this answers is not in the document]
Comment (dahlstrom-g) [reply third_party/crashpad/client/BUILD.gn:150]: It _is_ removed.

File: third_party/crashpad/third_party/googletest/BUILD.gn (hunk 1 above)
Comment (dahlstrom-g) [reply third_party/crashpad/third_party/googletest/BUILD.gn:1]: This is what we get for using `git-subtree.sh`.

File: third_party/crashpad/minidump/BUILD.gn (hunk 2 above)
Comment (dahlstrom-g) [reply third_party/crashpad/minidump/BUILD.gn:116]: You don't have to guess!
https://chromium.googlesource.com/chromium/src/+/84.0.4147.144/third_party/crashpad/crashpad/minidump/BUILD.gn#110

File: third_party/crashpad/client/BUILD.gn (hunk 3 above)
[the remark this answers is not in the document]
Comment (hlwarriner) [reply third_party/crashpad/client/BUILD.gn:150]: Ah, I see it now.

**Comment by dahlstrom-g** [comment]
> I just focused on the affected files that had Starboard/Cobalt customizations but didn't see anything concerning. I left a few small comments but otherwise this looks good.

Thanks!

> I think I just owe you instructions to test on raspi-2 and android (using our AOSP port).

done

**Review by hlwarriner — APPROVED** [review]

**Pull request merged**